malasakit-django/pcari/views.py

```diff
@@ -216,7 +215,6 @@ def view_wrapper(request, *args, **kwargs):
 def index(request):
     return redirect(reverse('pcari:landing'))
 
-
```

Comment:
Top-level functions should be separated by a [newline](https://www.python.org/dev/peps/pep-0008/#blank-lines), according to PEP8.